security/keycloak-oidc-client/src/test/resources/keycloak-realm.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
Is this file used? I saw just .yaml one referenced

Reply:
The `json` keycloak realm is used by `KeycloakQuarkusTestResource`.